Rock.Messaging/MQ/BinarySenderMessage.cs

```diff
@@ -49,7 +53,7 @@ public BinarySenderMessage(byte[] binaryValue, Encoding encoding = null, byte? p
         /// <param name="messageFormat">The message's format.</param>
         /// <param name="encoding">
         /// The encoding to use when converting the binary value to a string if 
-        /// <paramref name="messageFormat"/> is not <see cref="Messaging.MessageFormat.Binary"/>.
+        /// <paramref name="messageFormat"/> is not <see cref="Rock.Messaging.MessageFormat.Binary"/>.
```

Comment:
Does the `cref` for `MessageFormat.Binary` need to be namespace-qualified? As written, it won't link properly (and will create a compiler warning) for RockLib. If it's not namespace qualified, it should work fine.